dlt/destinations/impl/duckdb/configuration.py

```diff
@@ -86,6 +85,14 @@ def parse_native_representation(self, native_value: Any) -> None:
             else:
                 raise
 
+    @property
```

Comment:
much better, thanks :) you don't need to change this anymore it is good, but one feedback from me would be that you don't really need getter and setter here, it adds more coded that is not needed since this is not really a public interface used by anyone. adding the conn_config function alone and bulding the connection string there would be enough. But we can keep it like this.

Reply:
@sultaniman: sorry for another change request, maybe I was not clear in my comment. Using hasattr and getattr is not really preferred as we lose typing and it also really is not needed here. They way to leverage inheritance here is the following:
1. create a private function def _connection_config() in the superclass that returns the default value (empty dict or none, whatever you need there)
2. Leave the default behavior for duckdb (do not change anything there)
3. Overwrite this function in the motherduck implementation to return the correct dict (you do not need to prepare this dict in the init or anywhere, just build it there. less lines of code the better)
4. call connection_config() in the superclass when creating the connection.

Reply:
Thanks I think yeah it was a minor misunderstanding, I will adjust it.